resources/views/livewire/cache.blade.php

```diff
@@ -20,7 +20,7 @@
                     Str::plural('group', $count)
                 );
             @endphp
-            <button title="{{ $message }}" @click="alert('{{ str_replace("\n", '\n', $message) }}')">
+            <button title="{{ $message }}" @click="alert(@js($message))">
```

Comment:
Standardising use of the `@js` directive, which I've used in some of the changes in this PR.